.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,22 +11,36 @@ https://sunpy.org/coc
 Furthermore, you might need to check with your work place if you are allowed to contribute code:
 https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html
 
-Also these comments are hidden when you submit this github pull request.
-
 We have several automatic features that run when a pull request is open.
 They can appear daunting but do not worry about them!
 Here is a brief explanation of them:
 https://docs.sunpy.org/en/latest/dev_guide/contents/pr_review_procedure.html#continuous-integration
--->
 
-### Description
+- Please create an issue _before_ creating a Pull Request
+- Please use the following Git commit message style
+  - Use the future tense ("Adds feature" not "Added feature")
+  - Limit the first line to 72 characters or less
 
-<!--
-Provide a general description of what your pull request does.
+## ↑👆 DELETE above _before_ submitting 👆↑
 
-If the pull request closes any open issues you can add this.
-If you replace <Issue Number> with a number GitHub will automatically link it.
-If it doesn't, please remove the following line.
+---
+
+# Description
+
+<!--
+Please include a summary of the changes and which issue will be addressed
+ Please also include relevant motivation and context.
 -->
 
-Fixes #<Issue Number>
+# Checklist
```

Comment:
Not sure if this is the best idea, but its a thought?

Reply:
I'd say get rid of "My" at the beginning of each bullet, but I like the chcklist

Reply:
Done.